task1/js/app.js

```diff
@@ -0,0 +1,46 @@
+let totalZombies = 0;
+let deadZombies = 0;
+let smallZombies = 0;
+let madZombies = 0;
+let strongZombies = 0;
+
+const totalSelector = document.querySelector('.total-zombies');
+const deadSelector = document.querySelector('.dead-zombies');
+const smallSelector = document.querySelector('.small-zombies');
+const madSelector = document.querySelector('.mad-zombies');
+const strongSelector = document.querySelector('.strong-zombies');
+
+for (const i in zombiesData) {
```

Comment:
syntax like
`for (const i in zombiesData)`
checks the all properties in array

in this case this is not a problem
but if you want to iterate the array elements use a simple for:
`for (const i = 0; i < zombiesData.length; i++)`